pkgs/by-name/ma/mapserver/package.nix

```diff
@@ -81,6 +84,16 @@ stdenv.mkDerivation rec {
     "-DCMAKE_SKIP_BUILD_RPATH=ON"
   ] ++ lib.optional withPython "-DWITH_PYTHON=ON";
 
+  passthru.updateScript = writeScript "update.sh" ''
+    #! ${stdenv.shell}
+    set -o errexit
+    set -x
+    # mapserver tags are: rel-MAJOR-MINOR-PATCH
+    # occasionnally, there are rel-MAJOR-MINOR-PATCH-betaN
+    version=$(${common-updater-scripts}/bin/list-git-tags --url=https://github.com/mapserver/mapserver | sed -E -n  "s/rel-([0-9])+-([0-9])+-([0-9])+$/\1.\2.\3/p" | sort --version-sort --reverse | head -n1)
```

Comment:
I tried to reuse common stuff as much as I could. It's really the non-standard tag scheme that prevents us to directly use gitUpdater.

Some scripts uses these headers or equivalent:

```
#!/usr/bin/env nix-shell
#!nix-shell -i bash -p coreutils common-updater-scripts
```
Others do:

```
      PATH=${with pkgs; lib.makeBinPath [ common-updater-scripts coreutils curl gnugrep gnused jq nix ]}
```

and some others do as I did. I don't know what's best!